implants/lib/eldritch/src/assets/read_impl.rs

```diff
@@ -0,0 +1,26 @@
+use anyhow::Result;
+
+pub fn read(src: String) -> Result<String> {
+    let src_file_bytes = match super::Asset::get(src.as_str()) {
+        Some(local_src_file) => local_src_file.data,
+        None => return Err(anyhow::anyhow!("Embedded file {src} not found.")),
+    };
+    let mut result = String::new();
+    for byte in src_file_bytes.iter() {
+        result.push(*byte as char);
```

Comment:
this seems sooooooo slow lol, is there no faster way?

Reply:
Not sure this is faster.
little harder to read but:
```rust
let res = src_file_bytes.iter().map(|x| *x as u32).collect::<Vec<u32>>();
```